src/load-styles.js

```diff
@@ -0,0 +1,89 @@
+import { maxSatisfying } from 'es-semver';
+
+const CDN_HOST = 'https://components.clever-cloud.com';
+const CSS = 'application/javascript';
+const ONE_YEAR = 365 * 24 * 60 * 60;
+
+export async function handleLoadStylesRequest (httpApi) {
+
+  const versionsList = await httpApi.fetch(`versions-list.json`, CDN_HOST);
+  if (versionsList == null) {
+    return httpApi.sendResponse({ body: '', status: 500 });
+  }
+
+  const version = getVersion(versionsList, httpApi.getQueryParam('version'));
+
+  const depsManifest = await httpApi.fetch(`deps-manifest-${version.resolved}.json`, CDN_HOST);
+  if (depsManifest == null) {
+    return httpApi.sendResponse({ body: '', status: 500 });
```

Comment:
It's OK like that,

When depsManifest is null, could we have a distinction between:
* I can't fetch the response because of network issue
* I can't fetch the response because the version you're asking is not present

The latter would maybe conduct to sending a 404 with "Uknown version X.Y.Z" as body.

Reply:
In fact, you may already know if the version doesn't exist when `version.resolved` is `null`.

Reply:
Forget what I said, I fixed it :+1: